packages/block-editor/src/components/block-content-overlay/style.scss

```
@@ -18,24 +12,17 @@
 		z-index: z-index(".block-editor-block-content-overlay__overlay");
 	}
 
-	&:hover:not(.is-dragging-blocks),
-	&.parent-highlighted {
-		> .block-editor-block-content-overlay__overlay {
-			background: rgba(var(--wp-admin-theme-color--rgb), 0.1);
-			box-shadow: 0 0 0 $border-width var(--wp-admin-theme-color) inset;
-		}
+	&:hover:not(.is-dragging-blocks).overlay-active::before,
+	&.parent-highlighted.overlay-active::before {
+		background: rgba(var(--wp-admin-theme-color--rgb), 0.1);
+		box-shadow: 0 0 0 $border-width var(--wp-admin-theme-color) inset;
 	}
 
-	&.overlay-active:not(.is-dragging-blocks) {
-		*:not(.block-editor-block-content-overlay__overlay) {
-			pointer-events: none;
-		}
+	&.overlay-active * {
```

```suggestion
	&.overlay-active:not(.is-dragging-blocks) * {
```
I think we need to condition this part on blocks not currently being dragged.  Currently trying to drag blocks into the template part only shows available dropzones in the top level of the template part and not inside any of its children.

Previously, we disabled pointer events on the overlay element when dragging blocks to allow us to have all drag-and-drop interactions underneath it.  Since that element is gone, I think that functionality needs to be handled by not removing pointer events from the children when drag-and-drop is active.